app/downstream_payload.rb

```diff
@@ -79,7 +79,10 @@ def message_queue_presenter
     when "redirect" then redirect_presenter
     when "gone" then gone_presenter
     when "vanish" then vanish_presenter
-    else content_presenter
+    when "substitute" then vanish_presenter
```

Comment:
Should this not be `substitute_presenter`?

Reply:
Oh, I see the `substitute_presenter` method was added in a later commit.  Maybe this first commit should just add the logging, then the `when "substitute"` case could be added in the same commit as the `substitute_presenter` method?